.github/workflows/main-build-python38.yml

```diff
@@ -0,0 +1,93 @@
+name: Python38 Layer Integration Test
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-20.04
+    name: Python38-${{ matrix.architecture }}-IntegrationTest
+    strategy:
+      fail-fast: false
+      matrix:
+        architecture: [ amd64, arm64 ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17.7'
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_ARN }}
+          role-duration-seconds: 1200
+          aws-region: us-east-1
+      - name: Patch ADOT
+        run: ./patch-upstream.sh
+      - name: Login to Public ECR
+        uses: docker/login-action@v1
+        with:
+          registry: public.ecr.aws
+      - name: Build layers / functions
+        run: GOARCH=${{ matrix.architecture }} ./build.sh
+        working-directory: python
+      - name: Get Lambda Layer amd64 architecture value
+        if: ${{ matrix.architecture == 'amd64' }}
+        run: echo LAMBDA_FUNCTION_ARCH=x86_64 | tee --append $GITHUB_ENV
+      - name: Get Lambda Layer arm64 architecture value
+        if: ${{ matrix.architecture == 'arm64' }}
+        run: echo LAMBDA_FUNCTION_ARCH=arm64 | tee --append $GITHUB_ENV
+      - name: Get terraform directory
+        run: |
+          echo TERRAFORM_DIRECTORY=python/integration-tests/aws-sdk/wrapper |
+          tee --append $GITHUB_ENV
+      - uses: hashicorp/setup-terraform@v1
+      - name: Initialize terraform
+        run: terraform init
+        working-directory: ${{ env.TERRAFORM_DIRECTORY }}
+      - name: Get terraform Lambda function name
+        run: |
+          echo TERRAFORM_LAMBDA_FUNCTION_NAME=lambda-python-${{ matrix.architecture }} |
+          tee --append $GITHUB_ENV
+      - name: Apply terraform
+        run: terraform apply -auto-approve
+        working-directory: ${{ env.TERRAFORM_DIRECTORY }}
+        env:
+          TF_VAR_sdk_layer_name: opentelemetry-python-aws-sdk-wrapper-${{ matrix.architecture }}
+          TF_VAR_function_name: ${{ env.TERRAFORM_LAMBDA_FUNCTION_NAME }}
+          TF_VAR_architecture: ${{ env.LAMBDA_FUNCTION_ARCH }}
+          TF_VAR_runtime: 'python3.8'
```

Comment:
Is this the point we verify new building in Lambda python 3.8 environment?

Reply:
Tha's true, we are using the variable to pass the `runtime` for verifying.